server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1158,29 +1157,25 @@ long getCompactionCancelID() {
       }
 
       return new Pair<>(compactID, compactionConfig);
-    } catch (InterruptedException | DecoderException | NumberFormatException e) {
-      throw new RuntimeException(e);
+    } catch (IOException | InterruptedException | DecoderException | NumberFormatException e) {
+      throw new RuntimeException("Exception on " + extent + " getting compaction ID", e);
```

Comment:
I wonder if `UncheckedIOException` or `IllegalStateException` would be more appropriate here.

Reply:
Since we are catching 4 different ones, I think Runtime is OK.

Reply:
Yeah, it probably doesn't matter. I just like when we can group them into a larger category. A lot of times, you can do that easily with IllegalArgumentException, for example... even if there are multiple exceptions which can occur in processing, the end result as far as the caller is concerned, is that they passed in an improper argument. But here, it's not quite as clear.